### DISCUSSION TITLE
Add support for zlib-ng

### DISCUSSION THREAD

**Discussion by demhademha** [pete4abw/lrzip-next#62] [Ideas]
The [following](https://github.com/zlib-ng/zlib-ng) has come to my attention. 
Would the performance gained from zlib-ng be worth it for lrzip-next?

**Comment by pete4abw** [comment]
Probably not. Unless you used GZIP which no one does. In fact, I have never used gzip, bzip2, or lzo. It defeats the purpose of `lrzip`, doesn't it? If you can think of a good use case for it, it should not be difficult. But it has to be a 100% drop in.
